core/templates/dev/head/editor/state_editor_content.html

```diff
@@ -13,6 +13,8 @@
       </span>
       <span angular-html-bind="content[0].value" class="protractor-test-state-content-display"></span>
     </div>
+    <!-- This is a dummy divison created to mask the contents when hovoverd over above division -->
```

Comment:
(Actually, edit: s/division/div. I don't think anyone ever really calls that 'division'...)